### ISSUE TITLE
Flexbox properties trigger “Use known properties” warning

### ISSUE THREAD

**Issue by revoltpuppy** [CSSLint/csslint#543]
Using flexbox properties brings up the warning, ﻿﻿﻿“Require use of known properties”. Flexbox is now an official layout method.

**Comment by ArgTang** [comment]
Hello

Seems like the parser lib adds support for display: flex. there is a new version available, that might fix this. ref #563

**Comment by frankpinto** [comment]
The website http://csslint.net/ still reports "require use of known properties" when it encounters a line like the following:

```css
  flex: 1;
```

Is this expected behavior?

**Comment by serhiy-yevtushenko** [comment]
Are there plans to do the new release of CSSLint to incorporate the update in parserlib version [#563](https://github.com/CSSLint/csslint/issues/563)  in the CSSLint? The last release of CSSLint was done two years ago.

On the project where we are using CSSLint flex property is used in CSS, and we stay between dilemma either to switch off "known-properties" check or allow false positives for warning for flex properties. Having a new release would have solved this problem

**Comment by frvge** [comment]
@serhiy-yevtushenko you can now use the csslint allow structure (from master branch) to allow it. We will do an release of both parser-lib and csslint soon(tm).

**Comment by frvge** [comment]
I'm closing this as duplicate of #514 .
